src/lib/customers/20minuten-de.json

```diff
@@ -0,0 +1,54 @@
+	"logo": {
+		"file": "tx-group.webp",
+		"alt": "TX Group"
+	},
+	"customer": "<a href=\"https://www.20min.ch/\" target=\"_blank\">20 Minuten</a> ist die reichweitenstärkste Medienmarke der Schweiz. Sie gehört wie auch das Medienhaus Tamedia und die Werbevermarkterin Goldbach zur TX Group. 20 Minuten zeichnet sich durch ihre Reichweite in allen Sprachregionen und dem Storymix aus News, Unterhaltung und Inspiration aus und war/ist für bespinian ein wichtiger Partner in der Zusammenarbeit bezüglich der Migration der digitalen Produkte auf eine neue Infrastruktur, und anderen Projekten im Rahmen von OAuth2, Cloudifizierung, Containerisierung und Modernisierung von Software.",
+	"background": "20 Minuten ist die schweizweit grösste Pendlerzeitung und eine der meistbesuchten Webseiten der Schweiz. Es ist ein Medienportal mit täglichen Beiträgen aus dem Bereich News, Unterhaltung und Inspiration. 20 Minuten hatte 2019 eine grosse Migration ihrer Applikationen weg von der Legacy Infrastruktur und dem teilweise fast 20-jährigen Applikationscode geplant. Dabei ging es um eine Migration von bestehenden VMs auf eine moderne Kubernetes-basierte Containerinfrastruktur auf AWS. Das Backend sollte von Perl auf Go migriert und containerisiert werden. Das Frontend wurde ebenfalls von Perl gerendert und sollte in eine moderne Next.js Applikation umgewandelt werden. 20 Minuten hat täglich über eine Million Nutzer:innen. Dies macht eine Migration ohne Downtime besonders anspruchsvoll. Zusätzlich, war es wichtig, die Betriebskosten trotz dieses grossen Loads in einem überschaubaren Rahmen zu halten.",
+	"goal": "Das Ziel des Projekts war die Modernisierung des Setups der Applikation. Dabei sollten die Legacykomponenten und der frühere Code abgelöst werden, um das System stabiler und kosteneffizienter zu machen. AWS wurde als Infrastrukturprovider ausgewählt und die Applikation sollten in Go, Node.js und React Microservices auf einem EKS (Kubernetes) Cluster neu implementiert werden.",
+	"contribution": {
+		"intro": "bespinian hatte eine Schlüsselrolle in den folgenden Themen bezüglich der Applikationsmigration im Sinne von Refactoring / Re-architecting aus den 6 Migrationsstrategien von AWS:",
+		"topics": [
+			{
+				"title": "Infrastruktur auf AWS",
+				"body": "bespinian hat massgeblich zur Provisionierung der Infrastruktur auf AWS beigetragen. Dabei hatten wir eine Schlüsselrolle im Schreiben und Anwenden der entsprechenden Terraform-Module und im Aufsetzen der Kubernetes Cluster, auf denen die verschiedenen Microservices gehostet werden. Ein wichtiger Teil war dabei auch das korrekte Konfigurieren des Caching-Layers (Amazon CloudFront), um bei der grossen Nutzer:innenzahl von 20 Minuten die Kosten im überschaubaren Rahmen zu halten. bespinian hat viel von der technischen Umsetzung und Automatisierung, aber auch von der Kommunikation zwischen den Entwicklungs- und den Infrastruktur-Teams übernommen."
```

Comment:
Replace `um bei der grossen Nutzer:innenzahl von 20 Minuten die Kosten im überschaubaren Rahmen zu halten.` by `um bei der grossen Zahl von gleichzeitigen Nutzer:innen von 20 Minuten die Kosten in einem akzeptablen Rahmen zu halten.`